de.halaszovich.PMbrowser.json

```diff
@@ -0,0 +1,32 @@
+{
+  "app-id": "de.halaszovich.PMbrowser",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.3",
+  "sdk": "org.kde.Sdk",
+  "command": "QtPMbrowser",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host",
```

Comment:
this is a very broad filesystem permission.

Reply:
Okay, I will try to restrict the permission as much as possible. The user will want to load files and export data, not all of which can be can be handled by the portal mechanism.